uint8 switchState
----
uint32 retStat